impy/common.py

```diff
@@ -18,6 +18,28 @@
 from impy.util import info
 
 
+class EventData:
+    """Data structure to keep views of filtered data"""
+    def __init__(self, npart, p_ids, status, charge,
+                 px, py, pz, en, m, vx, vy, vz, vt,
+                 pem_arr, vt_arr):
+        self.npart = npart
+        self.p_ids = p_ids
+        self.status = status
+        self.charge = charge
+        self.px = px
+        self.py = py
+        self.pz = pz
+        self.en = en
+        self.m = m
+        self.vx = vx
+        self.vy = vy
+        self.vz = vz
+        self.vt = vt
+        self.pem_arr = pem_arr
+        self.vt_arr = vt_arr
```

Comment:
Should these be public? If not, use `self._pem_arr` and `self._vt_arr`. Please make everything private that is only for internal use. One can always later make private stuff public if necessary but vice versa is a breaking change.

Reply:
Yes, these arrays should be private for now. It will be changed.